### PR TITLE
SvmTrain

### PR DESCRIPTION
**Pull request by ben-albrecht**
Started `SvmTrain` - work-in-progress

- Reads in data
- Calls `calcKernel`
- Also updated Makefiles for pulling datasets

### CODE REVIEW

File: datasets/README.md [hunk 1]
```diff
@@ -5,7 +5,13 @@ datasets by executing `make` in this directory. You can also download
...
 
 ```bash
-make covtype.libsvm.binary
+make covtype
+```
+
+To see the available datasets, use:
+
+```bash
+make help
```
Comment (pgrew) [inline datasets/README.md:14]: I love this Makefile!

File: src/psvm/SvmTrain.chpl [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+use Documents;
+use Kernel;
+use Model;
+use Opt;
+
+// Enum
+use KernelType;
+
+
+             // TODO: Chapelify these names!
```
Comment (pgrew) [inline src/psvm/SvmTrain.chpl]: How do I Chapelify a name?

File: src/psvm/Kernel.chpl [hunk 3]
```diff
@@ -1,9 +1,62 @@
...
+  proc calcKernelWithLabel(a: Sample, b: Sample) {
+    var result = calcKernel(a, b);
+    if a.classLabel != b.classLabel then result *= -1.0;
+    return result;
+  }
+
+  proc innerProduct(a: Sample, b: Sample) {
+    var norm: real;
+
+    // TODO: Refactor as iterator:
```
Comment (pgrew) [inline src/psvm/Kernel.chpl:38]: Looks like a good exercise for me.

File: src/psvm/SvmTrain.chpl (hunk 2 above)
Comment (ben-albrecht) [reply src/psvm/SvmTrain.chpl]: Chapel tends to use `camelCase` naming for variables, functions, object instances, etc.

Comment (ben-albrecht) [reply src/psvm/SvmTrain.chpl]: There's no official Chapel style guide (today), but some of the stylistic guidelines are being discussed in [#6698](https://github.com/chapel-lang/chapel/issues/6698)

File: src/psvm/Kernel.chpl (hunk 3 above)
Comment (ben-albrecht) [reply src/psvm/Kernel.chpl:38]: Possibly! I'm happy to help you with that, if you're interested.

Comment (pgrew) [reply src/psvm/Kernel.chpl:38]: Would you mind pushing this branch?

**Pull request merged**